hammers/scripts/lease_stack_reaper.py

```diff
@@ -75,8 +78,8 @@ def find_stacked_leases(self, leases):
             else:
                 next_start_date = datetime.max
 
-            stacked_previous = (start_date - last_end_date).days < 1
-            stacked_next = (next_start_date - end_date).days < 1
+            stacked_previous = (start_date - last_end_date).days < 2
+            stacked_next = (next_start_date - end_date).days < 2
 
             if stacked_previous or stacked_next:
                 stacked.append(leases[i])
```

Comment:
You could make your life easier if this included the `node_type`

```python
stacked.append((node_type, leases[i][0]))
```

You only care about the `node_type` and `lease_id` from what I can tell. The other option would be having a thin object `Lease(id=, start_date=, end_date=, node_type=)` and just passing instances of that around. Sometimes a more explicit structure helps.

Reply:
I added in the node_type object in the print to help verify. It's not actually necessary unless we want to reference it in the email.